invenio_app_ils/ill/views.py

```diff
@@ -23,12 +23,18 @@
     patron_loan_extension_accept_loader, \
     patron_loan_extension_decline_loader, \
     patron_loan_extension_request_loader
+from .mail.tasks import send_ill_mail
 from .proxies import current_ils_ill
 
 
-def create_ill_actions_blueprint(app):
-    """Create ILL actions blueprint."""
-    blueprint = Blueprint("ils_ill_actions", __name__, url_prefix="")
+def create_ills_blueprint(app):
```

Comment:
Is this a typo, and if so which one the `l` or the `s` :) ? I know by its directory that it is `ill` but it could be `ils`.